package-lock.json

```diff
@@ -4,35 +4,1493 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "@types/node": {
```

Comment:
Did it just install a huge number of NPM packages here?

Reply:
Yes. Is it a problem?

Reply:
Nope. Was just wondering whether it was intended